script/changelogAnalysis/generateContractChangelog.ts

```
@@ -0,0 +1,356 @@
+async function mainWithAI() {
+  console.log('🤖 AI-powered analysis (Claude Sonnet)\n')
+  
+  const changedFiles = getChangedSolidityFiles()
+  
+  if (changedFiles.length === 0) {
+    console.log('ℹ️  No Solidity files changed')
+    return
+  }
+  
+  console.log(`Found ${changedFiles.length} changed contract(s):\n`)
+  changedFiles.forEach(file => console.log(`  - ${file}`))
```

Biome (2.4.4)

[error] This callback passed to forEach() iterable method should not return a value.

(lint/suspicious/useIterableCallbackReturn)